dbt/models/iasworld/schema/iasworld.comdat.yml

```diff
@@ -88,21 +88,35 @@ sources:
                   name: iasworld_comdat_chgrsn_eq_5_mktadj_not_null
                   expression: |
                     != '5' OR mktadj IS NOT NULL
+                  additional_select_columns:
+                    - parid
+                    - taxyr
+                    - card
+                    - who
+                    - wen
+                    - mktadj
```

Comment:
This test, along with `iasworld_comdat_class_in_ccao_class_dict` and `iasworld_dweldat_class_in_ccao_class_dict`, have started failing since the last time we published a workbook, so they need descriptions and additional select columns. I went ahead and bundled them into this PR, since this test makes use of the new format for `additional_select_columns` when using `expression_is_true`.